### DISCUSSION TITLE
Better view options

### DISCUSSION THREAD

**Discussion by strikingtwice** [WinMerge/winmerge#1803] [Q&A]
Hi everyone! Loving winmerge, on mac i used to use a program called kaleidoscope, and it was a little prettier, of course as most software is compared to the opensource stuff, but I was wondering if i could maybe get some suggestions on better view options? I only ever use it for file/folder comparison and syncing, and I was wondering if there was a cleaner more direct way of seeing. I've included a screenshot of kaleidoscope for reference, and you can see how there are blank spaces and that you can kinda see both folders at once in a little bit of a cleaner way, rather than reading the "left only/right only" columns. I searched the forum but it's either a bad search or the other items that showed up weren't relevant to what I needed.
![image](https://user-images.githubusercontent.com/5437866/232105815-4de3e9b6-129a-4cfe-9f9d-84648c2265e1.png)

**Comment by sdottaka** [comment]
Unfortunately there is no option to display like that image.

Similar discussion:
https://github.com/WinMerge/winmerge/discussions/868

**Comment by strikingtwice** [comment]
Thanks for the info!-David DeLizzaOn Apr 14, 2023, at 8:29 PM, Takashi Sawanaka ***@***.***> wrote:﻿
Unfortunately there is no option to display like that image.
Similar discussion:
#868
